### DISCUSSION TITLE
Other syntax to call ccode.

### DISCUSSION THREAD

**Discussion by ClientCrash** [SaptakBhoumik/Peregrine#43] [Ideas]
If i understand it correctly, your wrap your c function calls in this way : `Ccode printf("Hello World!"); Ccode`

I recommend a more practical and/or beautiful solution : `Ccode {
printf("Hello World");
}`

**Comment by SaptakBhoumik** [comment]
@ClientCrash We have made  ``` Ccode printf("Hello World!"); Ccode``` intentionally to prevent them from calling unsafe function too often

**Reply by ClientCrash** [reply]
But sometimes you have to call unsafe functions, for example in system development. But i understand the decision.

**Reply by SaptakBhoumik** [reply]
You can call unsafe function but the current syntax is to reduce the rate of unsafe function calls

**Comment by sairam4123** [comment]
I think this syntax could be better if this feature is added.
```py
unsafe:
   ... # your unsafe C-Code
```

**Reply by SaptakBhoumik** [reply]
Ccode is no longer supported because of the mess it creates. Use extern instead
